client/components/jetpack/card/jetpack-product-card/display-price/time-frame.tsx

```diff
@@ -98,20 +101,21 @@ const PartialDiscountTimeFrame: React.FC< PartialDiscountTimeFrameProps & A11yPr
 		count: discountedPriceDuration,
 		args: {
 			months: discountedPriceDuration,
+			original_price: formattedOriginalPrice,
 		},
 	};
 
 	/* eslint-disable wpcalypso/i18n-mismatched-placeholders */
 	let text = translate(
-		'for the first month, billed yearly',
-		'for the first %(months)d months, billed yearly',
+		'for the first month, then %(original_price)s /month billed yearly',
```

Comment:
Since these new strings are not yet translated (I think?), you might want to use `hasTranslation` to keep the old string for the short term (unless you plan to wait for this PR until the translations complete before merging). Example: https://github.com/Automattic/wp-calypso/pull/63977/files